### PR TITLE
Feat subtraction

### PR DESCRIPTION
**Pull request by mr-gilberto**
Created feature subtraction

Testing done: Pixel 2 API 27

Author: gilberto.ibarra@globant.com

### CODE REVIEW

File: app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public int getOperation() {
+        return operation;
+    }
+
+    public double doOperation(int operation, double a, double b) {
+        switch (operation) {
+            case Constants.SUM:
+                return a + b;
```
Comment (gaston-mira-glb) [inline app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java]: add break for each case

File: app/src/main/java/module2/globant/calculator/mvp/presenter/CalculatorPresenter.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        }
+
+        return validated;
+    }
+
+    public void register() {
+        Activity activity = view.getActivity();
+
+        if (activity == null) {
+            return;
```
Comment (gaston-mira-glb) [inline app/src/main/java/module2/globant/calculator/mvp/presenter/CalculatorPresenter.java]: it's better if you use 
```java 
if (activity != null) {
     RxBus.subscribe(activity, new OperationButtonBusObserver() 
```

The code is more legilable

File: app/src/main/java/module2/globant/calculator/mvp/presenter/CalculatorPresenter.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        });
+
+    }
+
+
+    public void unregister() {
+        Activity activity = view.getActivity();
+
+        if (activity == null) {
```
Comment (gaston-mira-glb) [inline app/src/main/java/module2/globant/calculator/mvp/presenter/CalculatorPresenter.java]: here the same

File: app/src/main/java/module2/globant/calculator/mvp/view/CalculatorView.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import butterknife.OnClick;
+import module2.globant.calculator.Constants.Constants;
+import module2.globant.calculator.R;
+import module2.globant.calculator.bus.RxBus;
+import module2.globant.calculator.bus.observers.OperationButtonBusObserver;
+import module2.globant.calculator.bus.observers.ResultButtonBusObserver;
+
+public class CalculatorView extends ActivityView {
+
+    @BindView(R.id.result_label)
```
Comment (gaston-mira-glb) [inline app/src/main/java/module2/globant/calculator/mvp/view/CalculatorView.java]: you can put this in the same line
```java
@BindView(R.id.result_label) TextView resultLabel;
```

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -8,15 +8,23 @@
...
     <TextView
+        android:id="@+id/operation_symbol_label"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:textColor="@color/colorAccent"
+        android:text="@string/main_plus_button"
+        android:textSize="18sp"/>
```
Comment (gaston-mira-glb) [inline app/src/main/res/layout/activity_main.xml]: use dimens please

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -69,17 +81,19 @@
...
 
     <TextView
+        android:id="@+id/result_label"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_gravity="center"
         android:gravity="center"
         android:text="@string/main_zero_hint"
-        android:textSize="@dimen/activity_vertical_margin" />
+        android:textSize="27sp" />
```
Comment (gaston-mira-glb) [inline app/src/main/res/layout/activity_main.xml]: here too

**Review by gaston-mira-glb — CHANGES_REQUESTED** [review]

File: app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java [hunk 7]
```diff
@@ -18,6 +18,8 @@ public double doOperation(double a, double b) {
         switch (operation) {
             case Constants.SUM:
                 return a + b;
+                case Constants.SUBTRACTION:
```
Comment (fanky10) [inline app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java]: please fix tab alignment

**Review by fanky10 — CHANGES_REQUESTED** [review]

File: app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java [hunk 8]
```diff
@@ -18,6 +18,8 @@ public double doOperation(double a, double b) {
         switch (operation) {
             case Constants.SUM:
                 return a + b;
+             case Constants.SUBTRACTION:
```
Comment (gaston-mira-glb) [inline app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java:21]: please fix tabulation

**Pull request merged**